apps/main-landing/src/app/creators/donate/content.tsx

```diff
@@ -279,17 +279,14 @@ export const Content = ({ className }: Properties) => {
       <Form onSubmit={formMethods.handleSubmit(onSubmit)} className="w-full">
         <Controller
           control={formMethods.control}
-          name="chainId"
+          name="tokenAddress"
```

Comment:
why the order of token/network was switched?

Reply:
Because of this comment
![Zrzut ekranu 2024-11-30 o 14 51 04](https://github.com/user-attachments/assets/f263dfc3-d0c3-4eeb-90c4-87d157654dda)

Reply:
okay, thanks 💪